sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java

```diff
@@ -537,6 +509,42 @@ void setSQLiteOpenHelper(@NonNull SQLiteOpenHelper helper) {
         mSQLiteOpenHelper = helper;
     }
 
+    /**
+     * Set maximum SQLite database size.
+     *
+     * @param maxStorageSizeInBytes Maximum SQLite database size.
+     * @return true if database size was set, otherwise false.
+     */
+    boolean setMaxStorageSize(long maxStorageSizeInBytes) {
```

Comment:
nit - to align with `public long getMaxSize()`, maybe we should can this `boolean setMaxSize(long maxStorageSizeInBytes)`. It's already in DatabaseManager class so no need to mention "storage".